### ISSUE TITLE
Flameshot cannot copy screenshot

### ISSUE THREAD

**Issue by ardishco-the-great** [flameshot-org/flameshot#3191]
### Flameshot Version

[when compiled from source]
Flameshot v12.1.0 (84acbdaf)
Compiled with Qt 5.15.9
[when using the flameshot-git package from the AUR]
Flameshot v12.1.0 (3ededae5)
Compiled with Qt 5.15.8

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...), User repository (AUR), Compiled from source

### Operating System type and version

6.3.1-arch1-1

### Description

This started happening after I compiled flameshot from source to try out some code changes in a PR (#3059) and afterwards, even running flameshot with the flameshot-git package, I could not copy anything.

I have tried
Compiling #3059 (has copy problem)
Compiling the master branch (has region select delay and copy problem)
Using the flameshot-git package in the AUR (has region select delay and copy problem)
Using the flameshot package in the community repo (doesn't run)

However, I have found that I can copy my screenshot If I:
1. Select Region
2. Upload image to imgur
3. Open Url (can't copy it because that doesn't work either)
4. Copy the image from the opened URL

This is better than before for sure, I couldn't even do that on the other packages after compiling flameshot from source for the first time

Things to note: I have all the dependencies installed, so I'm pretty sure it's not a problem with a dependency.

### Steps to reproduce

1. Install flameshot (can compile from source or use a provided package, doesn't matter)
2. Try to run flameshot gui
3. Select region
4. Copy
5. Try to paste
6. Realize you didn't actually copy anything in the first place

### Screenshots or screen recordings

![image](https://user-images.githubusercontent.com/109692107/236705648-fb5e0032-5c22-457b-9dea-a6b887aea3ee.png)

### System Information

Sway/Wayland

Dual monitors 2K 144hz/1080p 60hz

**Comment by NgoHuy** [comment]
I can copy/paste on Archlinux with sway. Do you install grim as dependency?

**Comment by ardishco-the-great** [comment]
> Things to note: I have all the dependencies installed, so I'm pretty sure it's not a problem with a dependency.

Yes, I have grim installed. The screenshot can be taken, just not copied, like I demonstrated with my alternative routes to doing so that I explained in my post.

**Comment by NgoHuy** [comment]
please remove .config/flameshot and .config/flameshotrc, it should work

**Comment by NgoHuy** [comment]
After testing, I can workaround by launching flameshot first, then use flameshot gui, it should copy. On KDE, it worked fine without launching first. 
Note that, with official package on Archlinux, it uses dbus by default, with git it should be built with GRIM option.
For developers, I saw the message when launching flameshot gui directly
![image](https://user-images.githubusercontent.com/1238112/236997220-4edfc8c4-af93-41e3-910d-f815d6bcc560.png)

**Comment by ardishco-the-great** [comment]
> Note that, with official package on Archlinux, it uses dbus by default, with git it should be built with GRIM option.

I built flameshot-git with the DUSE_WAYLAND_GRIM flag, edited the PKGBUILD and all that before using all that.

> please remove .config/flameshot and .config/flameshotrc, it should work with flameshot and left click icon on tray
Furthermore, I would like to add that this didn't change anything in my case.

**Comment by NgoHuy** [comment]
> > Note that, with official package on Archlinux, it uses dbus by default, with git it should be built with GRIM option.
> 
> I built flameshot-git with the DUSE_WAYLAND_GRIM flag, edited the PKGBUILD and all that before using it.
> 
> > please remove .config/flameshot and .config/flameshotrc, it should work with flameshot and left click icon on tray
> 
> Furthermore, I would like to add that this didn't change anything in my case.

did you run flameshot first, then flameshot gui?

**Comment by ardishco-the-great** [comment]
> 

Yes, I run `flameshot` at launch and when I press printscreen It runs `flameshot gui`

**Comment by ardishco-the-great** [comment]
I think there is a chance that compiling everything from scratch again might fix my issue, I would like to remove all the files, but I don't know where they all are.

Do you know where I can look for the files @NgoHuy ?
Or any other way I can uninstalling a compiled program?
